main.tf

```diff
@@ -216,6 +257,38 @@ resource "aws_appmesh_route" "integrators" {
   }
 }
 
+# resource "aws_appmesh_route" "test_route" {
```

Comment:
If these lines (which are in comment) are not necessary, please delete them.

Reply:
done